apps/files_reminders/src/components/SetCustomReminderModal.vue

```diff
@@ -0,0 +1,199 @@
+
+			<NcNoteCard v-if="isValid" type="info">
+				{{ t('files_reminders', 'We will remind you of this file') }}
+				<NcDateTime :timestamp="customDueDate" />
+			</NcNoteCard>
+
+			<NcNoteCard v-else type="error">
+				{{ t('files_reminders', 'Please choose a valid date & time') }}
+			</NcNoteCard>
+
+			<!-- Buttons -->
+			<div class="custom-reminder-modal__buttons">
+				<!-- Cancel pick -->
+				<NcButton @click="onClose">
+					{{ t('files_reminders', 'Cancel') }}
+				</NcButton>
+
+				<!-- Set reminder -->
+				<NcButton :disabled="!isValid" native-type="submit" type="primary">
+					{{ t('files_reminders', 'Set reminder') }}
+				</NcButton>
+			</div>
+		</form>
+	</NcModal>
+</template>
+
+<script lang="ts">
+import type { Node } from '@nextcloud/files'
+import { showError, showSuccess } from '@nextcloud/dialogs'
+import { translate as t } from '@nextcloud/l10n'
+import Vue from 'vue'
+
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
+import NcDateTime from '@nextcloud/vue/dist/Components/NcDateTime.js'
+import NcDateTimePickerNative from '@nextcloud/vue/dist/Components/NcDateTimePickerNative.js'
+import NcModal from '@nextcloud/vue/dist/Components/NcModal.js'
+import NcNoteCard from '@nextcloud/vue/dist/Components/NcNoteCard.js'
+
+import { getDateString, getInitialCustomDueDate } from '../shared/utils.ts'
+import { logger } from '../shared/logger.ts'
+import { setReminder } from '../services/reminderService.ts'
+
+export default Vue.extend({
+	name: 'SetCustomReminderModal',
+
+	components: {
+		NcButton,
+		NcDateTime,
+		NcDateTimePickerNative,
+		NcModal,
+		NcNoteCard,
+	},
+
+	data() {
+		return {
+			node: undefined as Node | undefined,
+			opened: false,
+			isValid: true,
+
+			customDueDate: getInitialCustomDueDate() as '' | Date,
+			nowDate: new Date(),
+		}
+	},
+
+	computed: {
+		fileId(): number {
+			return this.node.fileid
+		},
+
+		fileName(): string {
+			return this.node.basename
+		},
+
+		title() {
+			return t('files_reminders', 'Set reminder for "{fileName}"', { fileName: this.fileName })
+		},
+
+		label(): string {
+			return t('files_reminders', 'Set reminder at custom date & time')
+		},
+
+		clearAriaLabel(): string {
+			return t('files_reminders', 'Clear reminder')
+		},
+	},
+
+	methods: {
+		t,
+		getDateString,
+
+		/**
+		 * Open the modal to set a custom reminder
+		 * and reset the state.
+		 * @param node The node to set a reminder for
+		 */
+		async open(node: Node): Promise<void> {
+			this.node = node
+			this.isValid = true
+			this.opened = true
+			this.customDueDate = getInitialCustomDueDate()
+			this.nowDate = new Date()
+
+			// Focus the input and show the picker after the animation
+			setTimeout(() => {
+				const input = document.getElementById('set-custom-reminder') as HTMLInputElement
+				input.focus()
+				input.showPicker()
+			}, 300)
+		},
+
+		async setCustom(): Promise<void> {
+			// Handle input cleared
+			if (this.customDueDate === '') {
```

Comment:
```suggestion
			if (this.customDueDate === null) {
```
Since https://github.com/nextcloud-libraries/nextcloud-vue/pull/4540/commits/05d58f63c7f34558226fa248377693d8c3f09cbf